liblangutil/Exceptions.h

```diff
@@ -119,7 +119,9 @@ class Error: virtual public util::Exception
 		ParserError,
 		TypeError,
 		SyntaxError,
-		Warning
+		GeneralError,
+		Warning,
+		Notice
```

Comment:
Didn't we say to go with `Info` and note `Notice`?

Reply:
Yea but I started the PR before the decision, still need to swap the strings